addons/mail/static/src/core/common/data_response_model.js

```diff
@@ -20,6 +25,19 @@ export class DataResponse extends Record {
         return this.insert({ id: ++this._lastId });
     }
 
+    /** @override */
```

Comment:
Isn't it obvious that mail records override setup? What's the point of adding it as a jsdoc?